### PR TITLE
Feat/dog owners api

### PR DESCRIPTION
**Pull request by LileoC**
with endpoints to get all owners, one owner by id, delete owner, create owner, and update owner about me section.

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -16,5 +16,16 @@ class Woof(Base):
...
+
+class Dog_owner(Base):
+    __tablename__ = "dog_owners"
+
+    id = Column(Integer, primary_key=True, autoincrement=True, index=True)
+    username = Column(String, unique=True, index=True)
+    about_me = Column(String)
+    # email and password_hash should be added later for auth 
+    # email = Column(String)
+    # password_hash = Column(String)
```
Comment (wearethefoos) [inline app/models.py]: Better not commit commented out code

**Review by wearethefoos — APPROVED** [review]
Looks pretty good so far! Next time, try to commit one migration that has only the changes we eventually need. And please don't commit commented out code.

**Comment by LileoC** [comment]
@wearethefoos we got rid of the commented out code and the unnecessary migration version files. please let us know if it's ok to merge the pull request. :)

**Review by wearethefoos — APPROVED** [review]
Great! 🚢 🇮🇹

**Pull request merged**